presto-native-execution/presto_cpp/main/common/Counters.cpp

```diff
@@ -30,6 +30,27 @@ void registerPrestoCppCounters() {
       kCounterNumHTTPRequestError, facebook::velox::StatType::COUNT);
   REPORT_ADD_STAT_EXPORT_TYPE(
       kCounterHTTPRequestLatencyMs, facebook::velox::StatType::AVG);
+  REPORT_ADD_STAT_EXPORT_TYPE(
+      kCounterHttpClientPrestoExchangeNumOfOnBody,
+      facebook::velox::StatType::COUNT);
+  REPORT_ADD_HISTOGRAM_EXPORT_PERCENTILE(
+      kCounterHttpClientPrestoExchangeOnBodyBytes,
+      100,
+      0,
+      100000,
+      50,
+      90,
+      99,
+      100);
```

Comment:
Do we need count 100 percentile? @tanjialiang  @amitkdutta

Reply:
Remove 99 and add 95 to give us more data point; I remember when we do analysis on alluxio cache stats, p95 is actually pretty helpful

Reply:
> That literally means max. And why not? Maybe we can remove 99

A distributed system usually measure the long tail using P99 and P999 but no P100. We can have P100 but keep P99.